### DISCUSSION TITLE
Feature request: floating menu contents

### DISCUSSION THREAD

**Discussion by JimColl** [thegeeklab/hugo-geekdoc#368] [Ideas]
I love the theme, you've struck the prefect blend of features and layout without going overboard which is a hard mark to hit, kudos!

Hopefully it's within scope and not too much of a hassle to implement, but an option to allow the menu to float with the top of the screen as a user scrolls a page (as in the book theme, functionality demo'd @ (https://hugo-book-demo.netlify.app/docs/example/table-of-contents/)) would be the bees knees.  Thanks.

**Comment by xoxys** [comment]
Sorry for the way too long response time, looks like I missed the notification for this discussion. IM going to look into it and depending on the effort I'm fine to implement an option.
